Midterm/EchelonFour.py

```diff
@@ -0,0 +1,16 @@
+import time
+from Kill import Death
+
+class echelon_four(object):
```

Comment:
Python syntax rules and conventions want classes to be _CamelCase_ 

First letter capitalized, and other letters lower cased and then capitalizing words in lieu of spaces.

This example would be `class EchelonFour`

Also snake_case and lower case any file names